homeassistant/components/plugwise/__init__.py

```diff
@@ -74,6 +77,18 @@ async def async_update_data():
         except Smile.XMLDataMissingError as err:
             raise UpdateFailed("Smile update failed") from err
 
+    update_interval = timedelta(
+        seconds=entry.options.get(
+            CONF_SCAN_INTERVAL, DEFAULT_SCAN_INTERVAL["thermostat"]
+        )
+    )
+    if api.smile_type == "power":
+        update_interval = timedelta(
+            seconds=entry.options.get(
+                CONF_SCAN_INTERVAL, DEFAULT_SCAN_INTERVAL["power"]
+            )
+        )
```

Comment:
It would be nice if DEFAULT_SCAN_INTERVAL[api.smile_type] could be used here as well.

Reply:
Added both accordingly, good catch, tnx!